integration-tests/test/native-eth.spec.ts

```diff
@@ -40,6 +40,21 @@ describe('Native ETH Integration Tests', async () => {
     l2Bob = l1Bob.connect(env.l2Wallet.provider)
   })
 
+  describe('estimateGas', () => {
+    it('Should estimate gas for ETH transfer', async () => {
+      const amount = utils.parseEther('0.5')
+      const addr = await l2Bob.getAddress()
+      const gas = await env.ovmEth.estimateGas.transfer(addr, amount)
+      expect(gas).is.instanceof(BigNumber)
+    })
+
+    it('Should estimate gas for ETH withdraw', async () => {
+      const amount = utils.parseEther('0.5')
+      const gas = await env.ovmEth.estimateGas.withdraw(amount)
+      expect(gas).is.instanceof(BigNumber)
```

Comment:
We could add some more constraints here such as it being greater than zero?

Reply:
I'd be happy to add a check for greater than 0, but I didn't want to do any exact checks because that would be flakey over time. I was really trying to check to make sure that it didn't throw an error

Reply:
For any kind of gas snapshot tests we could use a pattern like the one here https://gist.github.com/gakonst/258a6ccd6bb752248999a13e8dd82a64